python_modules/libraries/dagster-components/dagster_components_tests/cli_tests/test_commands.py

```diff
@@ -93,23 +93,25 @@ def test_list_components_types_from_project() -> None:
         "definitions/local_component_sample",
         "definitions/other_local_component_sample",
         "definitions/default_file",
-    ) as tmpdir:
+    ) as (tmpdir, location_name):
```

Comment:
this is just working around weird importlib caching issues (that could not be solved with `importlib.invalidate_caches()`